README.dev.md

```diff
@@ -122,7 +123,7 @@ Install the following non-Python dependencies:
 ### On Debian or Ubuntu systems:
 
 ```
-sudo apt-get install libffi-dev memcached rabbitmq-server libldap2-dev python-dev redis-server postgresql-server-dev-all libmemcached-dev libfreetype6-dev
+sudo apt-get install closure-compiler libfreetype6-dev libffi-dev memcached rabbitmq-server libldap2-dev redis-server postgresql-server-dev-all libmemcached-dev python-dev hunspell-en-us nodejs nodejs-legacy npm python-virtualenv supervisor git yui-compressor puppet gettext
```

Comment:
virtualenv and supervisor aren't actually used in this installation process, so we can probably not include those.

The other missing sutff -- e.g. closure-compiler, gettext, and yui-compressor -- are only needed for building release tarballs.  We should probably include them in the centos/fedora instructions though.  Googling suggests the names are
```
yuicompressor
closure-compiler
gettext
```
so we should probably add those to the Fedora/CentOS lists as well...